vochain/genesis/genesis.go

```diff
@@ -14,8 +14,10 @@ var Genesis = map[string]Vochain{
 		SeedNodes: []string{
 			"7440a5b086e16620ce7b13198479016aa2b07988@seed.dev.vocdoni.net:26656",
 		},
-		CircuitsConfigTag: "dev",
-		Genesis:           &devGenesis,
+		Circuits: map[uint32]string{
+			0: "dev",
```

Comment:
I do think tags such as "dev" "prod" etc. Do not work anymore.
I'd just use standard versioning and Git tags.

Reply:
completely agree. funniest of all is that this whole Genesis.Circuits is read nowhere, is completely ignored

Reply:
i completely removed the "Tag" concept from circuit package, now everything refers to  "Version"